addons/website_slides/static/src/js/slides_course_tag_add.js

```diff
@@ -47,6 +47,17 @@ var TagCourseDialog = Dialog.extend({
         });
     },
 
+    /**
+     * Dirty hack to de-activate the "focustrap" from Bootstrap.
+     * Indeed, it prevents typing into our "select2" elements.
+     */
+    on_attach_callback: function () {
```

Comment:
this is annoying :/ shouldn't we make something generic for everybody ?

Reply:
Hello @std-odoo ! Sorry, I left this one on the side for a while. While the other PR is quite straightforward, this one is quite tricky. I did not find any other solution yet, and felt like this hack was better contained directly used in those two Dialogs. I'll try to find another solution with a rested head and come back to you :)